blog/2024-12-15-openhab-4-3-release.md

```diff
@@ -44,6 +44,62 @@ openHAB 4.3 brings XX new add-ons, adding support for new devices:
 
 But not only were new bindings added, some existing bindings also received massive improvements:
 
+### MQTT / Home Assistant
+
+The MQTT / Home Assistant binding allows integrating devices that communicate over MQTT, but were designed with Home Assistant in mind, rather than openHAB.
+Historically, openHAB's support for these devices has been buggy and missing many features, prompting many users to prefer to manually configure generic MQTT things rather than use auto-discovery, which can be time consuming and error prone.
+This release aims to make that practice no longer necessary, by fixing many bugs, and adding support for many missing features.
+
+The biggest change is the overall structure of Thing Type and Channel IDs.
+Whereas previously you may have had a Channel UID like `mqtt:homeassistant_zigbee2mqtt_5F0x8cf681fffe32e58e:mosquitto:zigbee2mqtt_5F0x8cf681fffe32e58e:0x8cf681fffe32e58e_5Fbattery_5Fzigbee2mqtt#sensor`, that same channel might now be `mqtt:homeassistant:mosquitto:zigbee2mqtt_5F0x00158d0007d3d7fa:battery`.
+Note that this release is transitional, and only newly discovered Things will use the new style IDs.
+You can opt in by deleting and re-creating your Things.
+Proper support for persisting dynamic Thing Types has been added, so the 2 minute delay before things would initialize after booting openHAB has been eliminated.
+There have been several issues fixed around keeping already created Things and Channels up to date if a device adds or removes components.
+The README has been re-written to give more useful details on what channels are supported for each component type.
+
```

Comment:
I considered adding a list of popular software that supports Home Assistant MQTT discovery, but thought that might be too much information.

Examples I use or am aware of:
 * zwave-js-ui
 * zigbee2mqtt
 * ESPHome
 * Theengs Gateway
 * OpenMQTTGateway
 * nodejs-poolController
 * Various other custom-built services

Reply:
I think adding a selection of these examples in a single sentence should be no problem and would be nice to have.